test/support/CustomMatchers.scala

```diff
@@ -388,13 +393,13 @@ object CustomMatchers {
   }
 
   def haveSelectionForPlaceOfDeposit(expectedSelectOptions: Seq[ExpectedSelectOption]): Matcher[Document] =
-    haveSelectionOptions("placeOfDeposit", "place of deposit", expectedSelectOptions)
+    haveSelectionOptions(FieldNames.placeOfDeposit, "place of deposit", expectedSelectOptions)
 
   def haveSelectionForOrderingField(expectedSelectOptions: Seq[ExpectedSelectOption]): Matcher[Document] =
-    haveSelectionOptions("field", "ordering field", expectedSelectOptions)
+    haveSelectionOptions(fieldKey, "ordering field", expectedSelectOptions)
 
```

Comment:
"note" and "background" to be replaced with field name in CustomMatchers.scala

Reply:
sorted!